Commands.adoc

```diff
@@ -0,0 +1,226 @@
+  # Set the following environment variables to enable access to the
+  # docker daemon running inside of the vagrant virtual machine:
+  export DOCKER_HOST=tcp://172.28.128.3:2376
+  export DOCKER_CERT_PATH=/foo/bar/.vagrant/machines/default/virtualbox/docker
+  export DOCKER_TLS_VERIFY=1
+  export DOCKER_API_VERSION=1.21
+
+  # run following command to configure your shell:
+  # eval "$(vagrant service-manager env docker)"
+----------------------------------------------------------------------------
+
+--
+*Note:* In the case of the docker service, the required TLS certificates for
+securing the Docker communication will be regenerated and copied to the host
+as part of the command execution. If existing certificates are found to be
+invalid, they get regenerated.
++
+--
+Similarly for OpenShift you can use: +
+`$ vagrant service-manager env openshift`
+
+--
+. To run the plugin and get the environment variables and certificates for
+docker in key=value format use:
++
+--
+----------------------------------------------------------------------
+$ vagrant service-manager env docker --script-readable
+DOCKER_HOST=tcp://172.28.128.3:2376
+DOCKER_CERT_PATH=/foo/bar/.vagrant/machines/default/virtualbox/docker'
+DOCKER_TLS_VERIFY=1
+DOCKER_API_VERSION=1.21
+----------------------------------------------------------------------
+--
+
+== Identify the IP Address and Version of the Running VM
+
+If you need to find information related to the running VM, such as the routable
+IP address or the version and release information use:
+
+`$ vagrant service-manager box` +
+
+Usage: `vagrant service-manager box [sub-command] [options]``
+
+The following sub-commands can be used: +
+`ip`: Displays the routable IP address of the running VM +
+`version`: Displays the version and release information about the running VM +
+
+The following options can be used: +
+`--script-readable` : Displays information in a key=value format +
+`-h, --help, print this help` : Prints help for the specific command being run.
+
+=== Examples
+
+. `$ vagrant service-manager box ip +
+172.28.128.3`
+
+. `$ vagrant service-manager box ip --script-readable +
+IP=172.28.128.3`
+
+. `$ vagrant service-manager box version +
+Atomic Developer Bundle (ADB) 2.2.0`
+
+. `$ vagrant service-manager box version --script-readable +
+VARIANT="Atomic Developer Bundle (ADB)" +
+VARIANT_ID="adb" +
+VARIANT_VERSION="2.2.0"`
+
+== Manage the lifecycle of a service configured in the running VM
+
+If you need to check the current status of existing services, start or stop or
+restart a specific service use: +
+`$ vagrant service-manager [subcommand] [service]`
+
+The following subcommands can be used: +
+`status`:   Lists services and their state (running/stopped) +
+`start`:   Starts the given service in the box +
+`stop`:   Stops the given service in the box +
+`restart`:   Restarts the given service in the box +
+
+The possible options for service are Docker, OpenShift and Kubernetes.
```

Comment:
the options are actually 'docker', 'openshift' and 'kubernetes'. Applies to all places where this is used. Docker, OpenShift and Kubernetes are the application names, but when talking about possible option values it is all lower case